kolibri/plugins/device/assets/src/views/FacilitiesPage/index.vue

```diff
@@ -285,6 +289,8 @@
           this.facilityForRemoval = facility;
         } else if (option === Options.REGISTER) {
           this.facilityForRegister = facility;
+        } else if (option === Options.MANAGESYNC) {
```

Comment:
Excellent, now you just need to add the `MANAGESYNC` key to the `OPTIONS` constant in line 194 of this file.

https://github.com/learningequality/kolibri/blob/07686f6764fb9b87b1c1c9c2f965c68e45b2631d/kolibri/plugins/device/assets/src/views/FacilitiesPage/index.vue#L194